featured-single-post.php

```diff
@@ -1,45 +1,72 @@
 <?php
 /**
  * Template Name: Featured Post
- * Template Post Type:
+ * Template Post Type: Post
  */
 disallow_direct_load( 'featured-single-post.php' );
 get_header( 'featured' ); the_post();
 
+$title = $post->post_title;
+$cats = wp_get_post_categories($post->ID);
+$category_title = get_cat_name($cats[0]);
 $subtitle = get_post_meta( $post->ID, 'subtitle', TRUE );
+$deck = get_post_meta( $post->ID, 'deck', TRUE );
+$source = get_post_meta( $post->ID, 'source', TRUE);
+$img_attach = get_img_html($post->ID, 'story_feature', Array('return_id' => True));
```

Comment:
It doesn't look like `$img_attach` or `$attachment` are used in this template; lines 15-19 can be removed.